### PR TITLE
Feat/oui datagrid domain

### PR DESCRIPTION
**Pull request by neolitec**

### CODE REVIEW

**Comment by jleveugle** [comment]
Quality check is already done ?

File: client/app/domain/tasks/TASKS.html [hunk 1]
```diff
@@ -1,71 +1,35 @@
...
+                    'label-danger': $row.status === 'ERROR',
+                    'label-default': $row.status === 'CANCELLED',
+                    'label-success': $row.status === 'DONE'
+                  }"
+                  data-ng-bind="tr('domain_tab_TASKS_status_' +  $row.status)">
+            </span>
+        </oui-column>
+        <oui-action-menu data-ng-if="$row.canCancel || $row.canRelauch || $row.canAccelerate"
+            data-align="end"
+            data-compact>
```
Comment (jleveugle) [inline client/app/domain/tasks/TASKS.html]: What is a compact button ? equivalent of xs in bootstrap ? I can't find a doc :( except "Use a compact button" ^^

File: client/app/domain/tasks/TASKS.html [hunk 2]
```diff
@@ -1,71 +1,35 @@
...
+            <oui-action-menu-item
+                data-text="{{ i18n.wizard_modify }}"
+                data-on-click="setAction('../domain-operation/update/domain-operation-update', $row)"></oui-action-menu-item>
```
Comment (jleveugle) [inline client/app/domain/tasks/TASKS.html]: Seems strange to not see ng-click but on-click in the context of angular :(

File: client/app/domain/glue/GLUE.html [hunk 3]
```diff
@@ -6,65 +6,21 @@
...
-                     data-items-per-page="itemsPerPage"
-                     data-transform-item="ctrlDomainGlue.transformItem(item)"
-                     data-on-transform-item-done="ctrlDomainGlue.onTransformItemDone(items)"
-                     data-page-placeholder="{{tr('pagination_page', [currentPage, nbPages])}}"
-                     data-item-per-page-placeholder="{{tr('pagination_display')}}"
-                     data-on-page-change="ctrlDomainGlue.loading = true"
-                     data-ng-show="!ctrlDomainGlue.loading && ctrlDomainGlue.glueDetails.length > 0">
-                </div>
-            </div>
+            <oui-datagrid data-rows="ctrlDomainGlue.glueHosts"
```
Comment (jleveugle) [inline client/app/domain/glue/GLUE.html]: Nice ;)

File: client/app/domain/tasks/TASKS.html [hunk 4]
```diff
@@ -1,71 +1,35 @@
...
+            data-title="tr('domain_tab_TASKS_function')"></oui-column>
+        <oui-column data-property="todoDate"
+            data-title="tr('domain_tab_TASKS_function')">
+            {{ $row.todoDate | date:'mediumDate' }}
+        </oui-column>
+        <oui-column data-property="comment"
+            data-title="tr('domain_tab_TASKS_comment')">
+            {{ $row.comment || '-' }}
+        </oui-column>
+        <oui-column property="status"
```
Comment (JDownloader) [inline client/app/domain/tasks/TASKS.html]: It's strange to use both property= and data-property=

Comment (neolitec) [reply client/app/domain/tasks/TASKS.html]: Yes!

File: client/app/domain/tasks/TASKS.html (hunk 1 above)
Comment (neolitec) [reply client/app/domain/tasks/TASKS.html]: Nope, it's the rounded button for action-menus. If the action-menu does not specifies this attribute, the button looks like a regular dropdown.

File: client/app/domain/tasks/TASKS.html (hunk 2 above)
Comment (neolitec) [reply client/app/domain/tasks/TASKS.html]: In all our components, we chose to use `on-click` ;)

**Comment by AxelPeter** [comment]
https://github.com/ovh-ux/ovh-manager-web/pull/267 has been merged. I change back the target branch to `develop`

**Pull request merged**